### DISCUSSION TITLE
r3f-spline export .splinecode not working

### DISCUSSION THREAD

**Discussion by chriscanin** [splinetool/r3f-spline#14] [Q&A]
The code sandbox environment and downloadable package work off of a .spline file in the examples provided, but export from the spline application as .splinecode. The .splinecode file is not able to be interpreted by the  "useSpline" function. This is the code in question.
`  const { nodes, materials } = useSpline(
    "https://prod.spline.design/gvgFxposolpeYZr8/scene.splinecode"
  );
  `
  
 And this is a link to my codesandbox project that was generated from a r3f export in the Spline App. 
 https://codesandbox.io/s/3-d-text-blue-copy-forked-g0hijd?file=/src/Scene.js:184-297
 
 PS: Fantastic work, I am excited this year to learn more 3d design and development with 3js and r3f, spline is the perfect tool to use, and work with. Thank you!

**Comment by ffdead** [comment]
Having the same issue. 
Here's my test export: https://codesandbox.io/s/spline-xr-test-9e67ti

**Comment by marcofugaro** [comment]
Text has some issue on codesandbox, try exporting to zip and working locally, should solve the issue.

<img width="234" alt="image" src="https://user-images.githubusercontent.com/7217420/212715533-117d9a05-1267-47ad-a78e-140530b6203a.png">

**Reply by ffdead** [reply]
Ok! 

Was hoping to set up a prototyping workflow for our designers to try their stuff in WebXR without too much dev-effort for each update. Is there some other way of viewing a spline scene in a WebXR session that you know of?
